ruby_review_week_four.rb

```diff
@@ -1,37 +1,135 @@
 
+
+
+
 # 7. What is handy about using gems in Ruby?
 
 # 8. Define a class called fish, and leave it empty.
+class Fish
+
+  def initialize(name, weight)
+    @name=name
+    @weight=weight
+  end
+end
+
+fash1=Fish.new("fash1","1kg");
+fash2=Fish.new("fash2","1.5kg");
+
+
+
 
 # 9. How would you use that class to make three fish objects?
 
+
+
+
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
+class Fish
+  attr_accessor :name :species
+  def initialize(name, species)
+    @name=name
+    @species=species
+  end
+end
+
 
 # 11. 
 
 # a. Create one fish object and store it in a variable, and 
+fish1=Fish.new("fish1","species1")
+fish2=Fish.new("fish2","species2")
+
 
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+fish1.name="Fred"
+
 # 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
 
+students={
+  name:heng,
+  age:24
+}
+
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
+def sum(num1,num2,num3)
+  sum=0
+  sum=num1+num2+num3
+  return sum
+  
+end
+sum(1,2,3)
+
 
 # 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
+def check(string,array)
+  if array.include?(string)
+  end
+end
+check("1",["1","2","3"])
+
+
 
 # 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise. 
 
-# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
\ No newline at end of file
+def decisible(number)
+  if number /3==0
+    return true
+  else
+    return false
+  end
+end
+
+p decisible(9)
+
+# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
+def decisible(number)
+  if number%3==0
+    return true
+  else
+    return false
+  end
+end
+
+def check(array)
+  array1=[]
+  array2=[]
+  array3=[]
+
+  array.each do |number|
+    if decisible(number)==true
```

Comment:
this looks correct, you shouldn't need the `== true` next to the method call, it will coerce to a true or false depending on the return of the method